config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: dws-test-driver-system
+namespace: dws-operator-system
```

Comment:
That causes problems.  When you do "make undeploy" you'll pull out this namespace from under DWS, and DWS will have a wreck.

I think you can go into your own namespace.

Reply:
I removed the namespace from the kustomize manifests, so this isn't an issue.